controllers/new-articles-controllers.js

```diff
@@ -0,0 +1,16 @@
+const { fetchAllArticles } = require('../models/new-articles-models');
+
+exports.getAllArticles = (req, res, next) => {
+    fetchAllArticles()
+      .then((articles) => {
+        const modifiedArticles = articles.map((article) => {
+          const { body, ...rest } = article;
+          return {
```

Comment:
This should all be done in the query to the DB. When possible we shouldn't be fetching all the data and then manipulating it, this should be done server side.